README.md

```diff
@@ -48,6 +48,33 @@ $ sudo dnf install mpris-scrobbler
 First, install the correct repository for your operating system from COPR.
 Then, install the `mpris-scrobbler` package with your package manager of choice.
 
+### Ubuntu 18.04
+
+Install the dependencies:
+
+    sudo apt install libevent-2.1-6 libevent-dev libdbus-1-dev dbus dbus-user-session libcurl4 libcurl4-openssl-dev libjson-c-dev libjson-c3 meson
+
+Restart after this to allow `dbus-user-session` to take effect.
```

Comment:
I don't think this step is required. Can you check if a 

    systemctl --user restart dbus.service 

would be enough?